WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Quick Start/QuickStartChecklistView.swift

```diff
@@ -0,0 +1,159 @@
+import UIKit
+import WordPressShared
+
+final class QuickStartChecklistView: UIView {
+
+    var tours: [QuickStartTour] = []
+    var blog: Blog?
+    var onTap: (() -> Void)?
+
+    private lazy var mainStackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [
+            labelStackView,
+            progressIndicatorView
+        ])
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .horizontal
+        stackView.alignment = .center
+        stackView.distribution = .fill
+        stackView.spacing = Metrics.mainStackViewSpacing
+        return stackView
+    }()
+
+    private lazy var labelStackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [
+            titleLabel,
+            subtitleLabel
+        ])
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.axis = .vertical
+        stackView.spacing = Metrics.labelStackViewSpacing
+        return stackView
+    }()
+
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = WPStyleGuide.serifFontForTextStyle(.body, fontWeight: .semibold)
+        label.textColor = .text
+        return label
+    }()
+
+    private lazy var subtitleLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = WPStyleGuide.fontForTextStyle(.callout)
+        label.textColor = .textSubtle
+        return label
+    }()
+
+    private lazy var progressIndicatorView: ProgressIndicatorView = {
+        let appearance = ProgressIndicatorView.Appearance(
+            size: Metrics.progressIndicatorViewSize,
+            lineColor: .primary,
+            trackColor: .listBackground
+        )
+        let view = ProgressIndicatorView(appearance: appearance)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        view.setContentHuggingPriority(.defaultHigh, for: .horizontal)
+        view.isAccessibilityElement = false
+        return view
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setupViews()
+        startObservingQuickStart()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("Not implemented")
+    }
+
+    deinit {
+        startObservingQuickStart()
+    }
+
+    func configure(tours: [QuickStartTour], blog: Blog, title: String, hint: String) {
+        self.tours = tours
+        self.blog = blog
+        titleLabel.text = title
+        accessibilityHint = hint
+        updateViews()
+    }
+}
+
+extension QuickStartChecklistView {
+
+    private func setupViews() {
+        addSubview(mainStackView)
+
+        NSLayoutConstraint.activate([
```

Comment:
Just a suggestion that you can ignore:

What about using `pinSubviewToAllEdges(_ subview: UIView, insets: UIEdgeInsets)` here?

Reply:
Fixed! [cf033d4](https://github.com/wordpress-mobile/WordPress-iOS/pull/18044/commits/cf033d4a49e7c9285fa207fcf91ebc2df25fb416)